package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "devDependencies": {
     "brunch": "^2.0.1",
+    "bower": "^1.6.0",
```

Comment:
It annoys me that we don't have the real required dependencies in `package.json`. This is the goal of this file...

Mainly, I think that anyone should be able to install all the dependencies with
``` bash
$ npm install
```

Using `-g` is an *advanced* way to do this.